### ISSUE TITLE
Question about accuracy calculation in test script

### ISSUE THREAD

**Issue by learnfromgroundup** [KevinMusgrave/pytorch-metric-learning#636]
Hi, I have just started learning PML and thanks for creating this great library!

I just have a stupid question to ask: in the test case of accuracy_calculator below, what is the element of knn_labels1? 
i believe each row is the prediction for each query, but what is the element in the row?
is it 
1. the i-th element is the label of i-th nearest neigbhour?
2. the i-th element is the the prediction of KNN for k=i?

```
        query_labels = torch.tensor([1, 1, 2, 3, 4, 0], device=TEST_DEVICE)

        knn_labels1 = torch.tensor(
            [
                [0, 1, 1, 2, 2],
                [1, 0, 1, 1, 3],
                [4, 4, 4, 4, 2],
                [3, 1, 3, 1, 3],
                [0, 0, 4, 2, 2],
                [1, 2, 3, 4, 5],
            ],
            device=TEST_DEVICE,
        )
        label_counts1 = ([0, 1, 2, 3, 4], [2, 3, 5, 4, 5])

        knn_labels2 = knn_labels1 + 5
        label_counts2 = ([5, 6, 7, 8, 9], [2, 3, 5, 4, 5])
    ```

**Comment by learnfromgroundup** [comment]
oops sorry, i think i have got the answer - it should be the i-th element is the label of i-th nearest neigbhour.
